openedx/adg/lms/conftest.py

```diff
@@ -47,3 +53,30 @@ def course_overviews(current_time):
         'test_course1': CourseOverviewFactory(language='en', **course_start_end_date),
         'test_course2': CourseOverviewFactory(language='ar', **course_start_end_date),
     }
+
+
+@pytest.fixture(name='prereq_course_groups')
+def create_prerequisite_course_group(request):
```

Comment:
Wouldn't it be better to choose another variable name instead of request, as it confuses with HTTP request, or at least define the variable in the docstring?

Reply:
Its a request fixture from pytest so we can't rename it but I'll add it in docstring